spdmlib/src/crypto/spdm_ring/asym_verify_impl.rs

```diff
@@ -118,7 +121,15 @@ fn asym_verify(
 }
 
 // add ASN.1 for the ECDSA binary signature
-fn ecc_signature_bin_to_der(signature: &[u8], der_signature: &mut [u8]) -> SpdmResult<usize> {
+fn ecc_signature_bin_to_der(
+    signature: &[u8],
+    der_signature: &mut [u8],
+    curve_algo: SpdmBaseAsymAlgo,
```

Comment:
I dont think we need to add curve_algo.
We just need to check sig.len() == NIST_P256_LEN or NIST_P384_LEN.

Reply:
> I dont think we need to add curve_algo. We just need to check sig.len() == NIST_P256_LEN or NIST_P384_LEN.

updated.